packages/commons/src/Deque.AxeCore.Commons.csproj

```diff
@@ -49,4 +49,18 @@
   <ItemGroup>
     <EmbeddedResource Include="node_modules/axe-core/axe.min.js" Link="Resources/axe.min.js" />
   </ItemGroup>
+
```

Comment:
This looks like a bad automatic merge resolution, the `DownloadAxe` target shouldn't be here twice